### PR TITLE
Disallow changing of role through EditCommand

### LINKED ISSUE #101: Remove the ability to edit Role of Person

### PR DESCRIPTION
**Pull request by Seonlo99**
fixes #101 
Disallow changing of role through EditCommand

- Edit command no longer accept r/ to change role.
- Role will not be allowed to change by any means after a person is created.
- Test cases are updated to not contain Role Prefix.

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/EditCommand.java [hunk 1]
```diff
@@ -57,7 +55,7 @@ public class EditCommand extends Command {
         "This person already exists in the contact list. Please use a different name.";
     public static final String MESSAGE_EDIT_ROLE_CONTAIN_AFFILIATION =
             "This person contains affiliations. Changing of Role is not allowed.";
-
+    public static final String MESSAGE_EDIT_ROLE_NOT_ALLOW = "Edit of role is not allow.";
```
Comment (shuenj) [inline src/main/java/seedu/address/logic/commands/EditCommand.java]: Dear sir, grammar sir

**Review by shuenj — APPROVED** [review]
LGTM, good work sir!

File: src/main/java/seedu/address/logic/commands/EditCommand.java (hunk 1 above)
Comment (Seonlo99) [reply src/main/java/seedu/address/logic/commands/EditCommand.java]: Opps... man's england not powderful enough hahaha, will change it in the next commit.

Comment (shuenj) [reply src/main/java/seedu/address/logic/commands/EditCommand.java]: thanks you sir

**Pull request merged**